sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -388,363 +388,47 @@ def get_transceiver_bulk_status(self):
-        sopmdlowwarning_flag         = BOOLEAN                          ; sopmd low warning flag
-        pdlhighalarm_flag            = BOOLEAN                          ; pdl high alarm flag
-        pdllowalarm_flag             = BOOLEAN                          ; pdl low alarm flag
-        pdlhighwarning_flag          = BOOLEAN                          ; pdl high warning flag
-        pdllowwarning_flag           = BOOLEAN                          ; pdl low warning flag
-        osnrhighalarm_flag           = BOOLEAN                          ; osnr high alarm flag
-        osnrlowalarm_flag            = BOOLEAN                          ; osnr low alarm flag
-        osnrhighwarning_flag         = BOOLEAN                          ; osnr high warning flag
-        osnrlowwarning_flag          = BOOLEAN                          ; osnr low warning flag
-        esnrhighalarm_flag           = BOOLEAN                          ; esnr high alarm flag
-        esnrlowalarm_flag            = BOOLEAN                          ; esnr low alarm flag
-        esnrhighwarning_flag         = BOOLEAN                          ; esnr high warning flag
-        esnrlowwarning_flag          = BOOLEAN                          ; esnr low warning flag
-        cfohighalarm_flag            = BOOLEAN                          ; cfo high alarm flag
-        cfolowalarm_flag             = BOOLEAN                          ; cfo low alarm flag
-        cfohighwarning_flag          = BOOLEAN                          ; cfo high warning flag
-        cfolowwarning_flag           = BOOLEAN                          ; cfo low warning flag
-        txcurrpowerhighalarm_flag    = BOOLEAN                          ; txcurrpower high alarm flag
-        txcurrpowerlowalarm_flag     = BOOLEAN                          ; txcurrpower low alarm flag
-        txcurrpowerhighwarning_flag  = BOOLEAN                          ; txcurrpower high warning flag
-        txcurrpowerlowwarning_flag   = BOOLEAN                          ; txcurrpower low warning flag
-        rxtotpowerhighalarm_flag     = BOOLEAN                          ; rxtotpower high alarm flag
-        rxtotpowerlowalarm_flag      = BOOLEAN                          ; rxtotpower low alarm flag
-        rxtotpowerhighwarning_flag   = BOOLEAN                          ; rxtotpower high warning flag
-        rxtotpowerlowwarning_flag    = BOOLEAN                          ; rxtotpower low warning flag
-        rxsigpowerhighalarm_flag     = BOOLEAN                          ; rxsigpower high alarm flag
-        rxsigpowerlowalarm_flag      = BOOLEAN                          ; rxsigpower low alarm flag
-        rxsigpowerhighwarning_flag   = BOOLEAN                          ; rxsigpower high warning flag
-        rxsigpowerlowwarning_flag    = BOOLEAN                          ; rxsigpower low warning flag
-        ================================================================================
+           Dictionary
```

Comment:
@mihirpat1 please add comment how this is different from status_flag fun

Reply:
@prgeor Addressed this now